wiki/pages/various/passages.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: default
+title: Example Passages
+parent: Big List of Stuff
+nav_order: 10
+has_children: false
+nav_exclude: false
+---
+{% include global.html %}
+<details closed markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+{: .text-delta }
+1. TOC
+{:toc}
+</details>
+# Example Passages
+Speaking is an important part of training voice. In addition to it being the main focus of vocal training for voice feminization (and masculization), attempting speech *during training* also gives us an insight as to how well we can maintain control of our vocal parameters in more complicated scenarios. We do this, usually, by reading passages.
```

Comment:
> voice feminization (and masculization)

Should replace with "voice gender modification".
> in more complicated scenarios

I'd disagree, it's more that we use passages so we don't have to put ourselves in actual conversation yet. It's a way of testing the voice and getting basic practice in.
> We do this, usually, by reading passages.

Passages are I would argue the second least effective way of training general speech, since each passage carries a certain stylistic component that encourages us to read it in a certain speech pattern. For some passages this is less present, and the rainbow passage is a good example of how something can be read as speech or as a script.

It might be best to just say something like "Here are some things to read! Getting actual conversation practice is important, but sometimes we just want a script to try things out."

Reply:
I might shrink this whole section down a bit using this feedback. I wasn't entirely sure what to put. Thank you!